golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
Hi @marguerite , nobody ever meant to offend or disregard your work. We are grateful for all the job you and the community did at packaging Go.

I might be wrong, but it looks like this change would cause all the kind of Go packages to require the Go compiler at runtime. This might be fine for static and shared libraries, but it would be useless for Go programs.

Would it be possible to rework this patch to avoid this issue?

As for the big picture about Go packaging, I'm currently writing a response to https://github.com/openSUSE/golang-packaging/issues/4 but I'm being slowed down by other tasks.